keepalived/cluster.sls

```diff
@@ -43,6 +43,9 @@ keepalived_{{ instance_name }}_notify:
 
 {%- for name, script in cluster.get('vrrp_scripts', {}).iteritems() %}
 keepalived_vrrp_script_{{ name }}:
+{%- if script.name is defined %}
+{%- set name=script.name %}
```

Comment:
What if more checks is using same script? Then each will override the same file, right?